DMCompiler/DMStandard/Types/Matrix.dm

```diff
@@ -36,6 +36,26 @@
 		f += Matrix2.f
 		return src
 
+	proc/Invert()
+		var/determinant = a*e - d*b
+		if(!determinant)
+			CRASH("Invalid matrix")
```

Comment:
Is this intended or does this also exist in Byond too? Fairly sure the official documentation says the matrix won't be changed by this proc when the matrix is not invertible.
```suggestion
			return src
```

Reply:
My implementation seems consistent with BYOND's behavior. It returns null if `Invert` runtimes, leaving the matrix itself unchanged.